### PR TITLE
Fix/doc 4253

### PR DESCRIPTION
**Pull request by mzorai**

### CODE REVIEW

**Comment by mzorai** [comment]
Content of this PR could be viewed through this link: https://docs-preprod.exoplatform.org/en/preproduction/Configuration.html#secure-access-to-webdav-urls-of-jcr-folders

File: docs/Configuration.rst [hunk 1]
```diff
@@ -64,6 +64,9 @@ Configuration
 
     -  :ref:`WebDAV configuration <Configuration.WebDAV>`
        Configuration of the WebDAV service.
+       
+    -  :ref:`Secure access to WebDAV urls of JCR folders <Configuration.SecureAccessWebdavJCR>`
```
Comment (ahamdi) [inline docs/Configuration.rst]: I prefer the following title : 
```suggestion
    -  :ref:`Secure the listing of the contents of JCR folders through Webdav <Configuration.SecureAccessWebdavJCR>`
```

File: docs/Configuration.rst [hunk 2]
```diff
@@ -64,6 +64,9 @@ Configuration
...
+       How to secure the access to WebDAV urls of JCR folders.
```
Comment (ahamdi) [inline docs/Configuration.rst]:
```suggestion
       How to secure the listing of the contents of folders through Webdav URLs.
```

**Review by ahamdi — CHANGES_REQUESTED** [review]
Could you check that we are speaking about allowing the listing of the contents for predefined URLs

File: docs/Configuration.rst [hunk 3]
```diff
@@ -2276,6 +2279,38 @@ The embedded WebDAV server lets you configure some parameter via :ref:`exo.prope
...
+
+==================================================================
+Secure the listing of the contents of JCR folders through Webdav
+==================================================================
+
+For security reasons, it is important to an administrator to secure the 
+access to WebDAV urls of JCR folders.
+
+A parameter ``exo.webdav.folder.listing.paths.allowed.regex`` is introduced 
+in eXo Platform allowing to define the set of WebDAV urls to JCR folders
```
Comment (ahamdi) [inline docs/Configuration.rst]:
```suggestion
in eXo Platform allowing listing the contents of JCR folders when they are accessed with Webdav 
```

File: docs/Configuration.rst [hunk 4]
```diff
@@ -2276,6 +2279,38 @@ The embedded WebDAV server lets you configure some parameter via :ref:`exo.prope
...
+==================================================================
+
+For security reasons, it is important to an administrator to secure the 
+access to WebDAV urls of JCR folders.
+
+A parameter ``exo.webdav.folder.listing.paths.allowed.regex`` is introduced 
+in eXo Platform allowing to define the set of WebDAV urls to JCR folders 
+to be listed.
+
+This parameter defines a regular expression of the allowed WebDav urls
```
Comment (ahamdi) [inline docs/Configuration.rst]:
```suggestion
This parameter defines a regular expression for the folders that could be listed  
```

File: docs/Configuration.rst [hunk 5]
```diff
@@ -2276,6 +2279,38 @@ The embedded WebDAV server lets you configure some parameter via :ref:`exo.prope
...
+==================================================================
+Secure the listing of the contents of JCR folders through Webdav
+==================================================================
+
+For security reasons, it is important to an administrator to secure the 
+access to WebDAV urls of JCR folders.
+
+A parameter ``exo.webdav.folder.listing.paths.allowed.regex`` is introduced 
+in eXo Platform allowing to define the set of WebDAV urls to JCR folders 
+to be listed.
```
Comment (ahamdi) [inline docs/Configuration.rst]: to be removed

File: docs/Configuration.rst [hunk 6]
```diff
@@ -2276,6 +2279,38 @@ The embedded WebDAV server lets you configure some parameter via :ref:`exo.prope
...
+
+This parameter defines a regular expression of the allowed WebDav urls 
+of JCR folders:
```
Comment (ahamdi) [inline docs/Configuration.rst]: to be removed

File: docs/Configuration.rst [hunk 7]
```diff
@@ -2276,6 +2279,38 @@ The embedded WebDAV server lets you configure some parameter via :ref:`exo.prope
...
+::
+
+		exo.webdav.folder.listing.paths.allowed.regex=(collaboration:/Users/(.*)/(.*)/(.*)/(.*))|(collaboration:/Groups/(.*))|(collaboration:/sites/(.*))|(portal-system:/production/app:gadgets/(.*))
+
+
+The above example allows the listing access to theses folders: users folders, 
+groups folders, sites folders and gadgets folders.
+
+.. note:: The value of the parameter ``exo.webdav.folder.listing.paths.allowed.regex``
+          should have this form: **wokspace_Name:/regex**.
```
Comment (ahamdi) [inline docs/Configuration.rst]:
```suggestion
          should respect this pattern : **wokspace_Name:/regex**.
```

File: docs/Configuration.rst [hunk 8]
```diff
@@ -2276,6 +2279,38 @@ The embedded WebDAV server lets you configure some parameter via :ref:`exo.prope
...
+          
+The default value of ``exo.webdav.folder.listing.paths.allowed.regex`` is set to empty 
+which means that **the access is allowed to all resources**.
```
Comment (ahamdi) [inline docs/Configuration.rst]:
```suggestion
which means that **the contents of All JCR folders are listed**.
```

**Review by ahamdi — CHANGES_REQUESTED** [review]
Some more comments :)

File: docs/Configuration.rst [hunk 9]
```diff
@@ -2276,6 +2279,38 @@ The embedded WebDAV server lets you configure some parameter via :ref:`exo.prope
...
+==================================================================
+Secure the listing of the contents of JCR folders through Webdav
+==================================================================
+
+For security reasons, it is important to an administrator to secure the 
+access to WebDAV urls of JCR folders.
+
+A parameter ``exo.webdav.folder.listing.paths.allowed.regex`` is introduced 
+in eXo Platform allowing listing the contents of JCR folders when they are accessed with Webdav 
+to be listed.
```
Comment (ahamdi) [inline docs/Configuration.rst]:
```suggestion

```

File: docs/Configuration.rst [hunk 10]
```diff
@@ -2276,6 +2279,38 @@ The embedded WebDAV server lets you configure some parameter via :ref:`exo.prope
...
+
+This parameter defines a regular expression for the folders that could be listed  
+of JCR folders:
```
Comment (ahamdi) [inline docs/Configuration.rst]:
```suggestion
```

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: docs/Configuration.rst (hunk 6 above)
Comment (mzorai) [reply docs/Configuration.rst]: i changed the sentence

**Review by ahamdi — APPROVED** [review]
LGTM

**Pull request merged**